.github/workflows/continuous-monitoring.yml

```diff
@@ -32,7 +32,11 @@ jobs:
           
       - name: Run smoke tests
         id: distribution-availability
-        run: ./gradlew :smoke-tests:check -PtestDistributionChannel=true
+        uses: nick-invision/retry@v2
```

Comment:
We should avoid random third party actions if possible, especially in a distribution channel monitor. I think we can update the cache action to cache dependencies as well instead

Reply:
Github action may occur downloading failure in anywhere, not only gradle.zip, if we cannot predict which part would get failure it is hard to add cache, now we just see an error then fix it. Also, adding more cache will inflate CI/CD code.

When we testing if the target binary/dependency is available, say xray-java-sdk in this case, we cannot add cache for it. Retry might be the only way to mitigate false download failure.